shaders/src/getters.vs

```diff
@@ -18,7 +18,11 @@ mat4 getLightFromWorldMatrix() {
 }
 
 PerRenderableData getObjectUniforms() {
+#if defined(MATERIAL_HAS_INSTANCES)
```

Comment:
This probably warrants a comment (and getters.fs) since it's pretty counter intuitive :)